src/server/router/application.ts

```diff
@@ -30,7 +27,65 @@ const TypeFormSubmissionSocial = z.object({
   socialLinks: z.string().nullish(),
 });
 
-type TypeFormSubmissionSocial = z.infer<typeof TypeFormSubmissionSocial>;
+export type TypeFormSubmissionSocial = z.infer<typeof TypeFormSubmissionSocial>;
+
+const TypeFormResponseField = z.object({
+  field: z.object({
+    id: z.string(),
+    type: z.string(),
+    ref: z.string(),
+  }),
+  type: z.string(),
+  text: z.string().nullish(),
+  date: z.date().nullish(),
+  file_url: z.string().nullish(),
+  boolean: z.boolean().nullish(),
+  phone_number: z.string().nullish(),
+  email: z.string().email().nullish(),
+});
+
+export type TypeFormResponseField = z.infer<typeof TypeFormResponseField>;
+
+const TypeFormResponseItems = z.array(
+  z.object({
+    landing_id: z.string(),
+    token: z.string(),
+    response_id: z.string(),
+    landed_at: z.date(),
+    submitted_at: z.date(),
+    metadata: z.object({
+      user_agent: z.string(),
+      platform: z.string(),
+      referer: z.string(),
+      network_id: z.string(),
+      browser: z.string(),
+    }),
+    hidden: z.object({
+      bobthebuilder: z.string(),
+    }),
+    calculated: z.object({
+      score: z.number(),
+    }),
+    answers: z.array(TypeFormResponseField),
+  })
+);
+
+export type TypeFormResponseItems = z.infer<typeof TypeFormResponseItems>;
+
+const TypeFormResponse = z.object({
+  total_items: z.number(),
+  page_count: z.number(),
+  items: TypeFormResponseItems,
+});
+
+export type TypeFormResponse = z.infer<typeof TypeFormResponse>;
+
+const options = {
+  method: "GET",
+  headers: {
+    Authorization: `Bearer ${env.TYPEFORM_API_KEY}`,
```

Comment:
We should not do this, and instead use the env helpers that T3 provides

Reply:
By this do you mean changing the import path from "process" to "../.../env/server.mjs"?

Reply:
<img width="390" alt="image" src="https://github.com/deltahacks/portal/assets/71908175/0399783a-080c-4282-a756-b22a747d3cd2">
I also found many more locations